jetty-ee11/jetty-ee11-tests/jetty-ee11-test-integration/src/test/java/org/eclipse/jetty/ee11/test/support/StringUtil.java

```diff
@@ -13,101 +13,16 @@
 
 package org.eclipse.jetty.ee11.test.support;
 
-import java.io.BufferedReader;
-import java.io.IOException;
-import java.io.StringReader;
-import java.util.ArrayList;
 import java.util.List;
 import java.util.ListIterator;
 
-import org.eclipse.jetty.util.IO;
+import org.eclipse.jetty.http.HttpTokens;
 
 public class StringUtil
```

Comment:
Should file another issue to see about removing this entirely and using the default jetty-util classes instead.

Reply:
It is not used. I'll remove